package.json

```diff
@@ -101,6 +101,7 @@
     "gh-pages": "^2.0.1",
     "glob": "^7.1.1",
     "jest": "^26.6.3",
+    "ky": "^0.12.0",
```

Comment:
what is this?

Reply:
answered over in this pr: https://github.com/cerner/terra-core/pull/3475/files#r684929622